static/js/screens/results.js

```diff
@@ -6,258 +6,340 @@
+        },
+
+        onLogout() {
+            this.reset();
+            this.screen.overlay({
+                icon: "exclamation",
+                title: "Bạn Chưa Đăng Nhập",
+                description: `Hãy đăng nhập vào CTMS để xem nội dung này!`,
+                buttons: {
+                    login: { text: "ĐĂNG NHẬP", icon: "signin", onClick: () => core.account.clickable.active = true }
+                }
+            });
+
+            this.screen.loading = false;
+        },
+
+        async load() {
+            if (!core.account.loggedIn) {
+                this.onLogout();
+                return;
+            }
+
+            try {
+                this.screen.loading = true;
+                await api.results();
+                this.screen.loading = false;
+            } catch (e) {
+                let error = parseException(e);
+
+                this.reset();
+                this.screen.overlay({
+                    icon: "bomb",
+                    title: "Toang Rồi Ông Giáo Ạ!",
+                    description: `<pre class="break">[${error.code}] >>> ${error.description}</pre>`,
+                    buttons: {
+                        login: { text: "THỬ LẠI", color: "pink", icon: "reload", onClick: () => this.load() }
+                    }
+                });
+
+                this.screen.loading = false;
+            }
+        },
+
+
+
+        /**
+         * 
+         * @param	{*}			param0 
+         * @returns {String[]}	array day of 2 week // format :2021-12-06
+         */
+
+        getScanDates(year, semester, count) {
+            var DateArr = []
+            if (semester === 1) {
+                var Databegin = new Date(`October 1, ${year}`);
+                Databegin = new Date(Databegin.setDate(Databegin.getDate() - Databegin.getDay() + 1));
+                for (let i = 0; i < count; ++i) {
+                    DateArr = [...DateArr, Databegin];
+                    Databegin = new Date(Databegin.setDate(Databegin.getDate() + 7));
+                }
+            } else if (semester === 2) {
+                var Databegin = new Date(`January 12, ${year}`);
+                Databegin = new Date(Databegin.setDate(Databegin.getDate() - Databegin.getDay() + 1));
+                DateArr = [...DateArr, Databegin];
+                Databegin = new Date(`March 3, ${year}`);
+                Databegin = new Date(Databegin.setDate(Databegin.getDate() - Databegin.getDay() + 1));
+                DateArr = [...DateArr, Databegin];
+            } else {
+                var Databegin = new Date(`July 7, ${year}`);
+                Databegin = new Date(Databegin.setDate(Databegin.getDate() - Databegin.getDay() + 1));
+                for (let i = 0; i < count; ++i) {
+                    DateArr = [...DateArr, Databegin];
+                    Databegin = new Date(Databegin.setDate(Databegin.getDate() + 7));
+                }
+            }
+            return DateArr
```

Comment:
Variable naming should follow Camel Case naming rule. For example, `DateArr` -> `dateArr`.
Please refer to [this](https://topdev.vn/blog/quy-chuan-dat-ten-trong-lap-trinh-camelcase-underscore-hay-pascalcase/).